docs/resources/projecttagsassociation.md

```diff
@@ -0,0 +1,82 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "rafay_projecttagsassociation Resource - terraform-provider-rafay"
+subcategory: ""
+description: |-
+  
+---
+
+# rafay_projecttagsassociation (Resource)
+
+This is an example of a project tag association.
+
+## Example Usage
+
+```terraform
+resource "rafay_project_tags_association" "tfptagassociation1" {
+  metadata {
+    name = "tfptagassociation1"
+    project = "terraform"
+  }
+  spec {
+    associations {
+      tag_key  = "tfkey1"
+      tag_type = "Cost"
+    }
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Argument Reference
+
+### Required
+
+- `metadata` - (Block List, Max: 1) Contains data that helps uniquely identify the resource. (See [below for nested schema](#nestedblock--metadata))
+- `spec` - (Block List, Max: 1) Defines the characteristics for the resource. This is the desired state for the resource. (See [below for nested schema](#nestedblock--spec))
+
+### Optional
+
+- `timeouts` - (Block) Sets the duration of time the create, delete, and update functions are allowed to run. If the function takes longer than this, it is assumed the function has failed. The default is 10 minutes. (See [below for nested schema](#nestedblock--timeouts))
+
+<a id="nestedblock--metadata"></a>
+### Nested Schema for `metadata`
+
+***Required***
+
+- `name` - (String) The name of the project tag association. This must be unique in your organization.
+- `project` - (String) The name of the Rafay project the cluster will be created in.
```

Comment:
The name of the Rafay project the cluster will be created in should be changed to The name of the Rafay project the project tag association will be created in